Gemfile

```diff
@@ -51,8 +51,15 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
-gem 'bootstrap', '~> 4.0.0.alpha6'
 gem 'simple_form'
+
```

Comment:
The particular version of the bootstrap gem we're using requires a few different gems to be included.  Generally, gems are supposed to add these dependencies in automatically, but the twitter bootstrap gem maintainers made us manually include these for this particular version of the gem.